### PR TITLE
Apps: Fixed missing 'targetLayer' attribute on grafana cmfpackage.json

### PR DESCRIPTION
**Pull request by obimelo**
This fix solves some missing attributes concerning the Apps:
- Fixed missing "targetLayer" attribute on grafana cmfpackage.json
- Fixed missing "usage" attribute on app deployment manifest

Unit tests results:
image

### CODE REVIEW

File: cmf-cli/resources/template_feed/grafana/Grafana.Package/cmfpackage.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "<%= $CLI_PARAM_CustomPackageVersion %>",
   "description": "Cmf Custom <%= $CLI_PARAM_Tenant %> <%= $CLI_PARAM_CustomPackageName %> Package",
   "packageType": "Grafana",
+  "targetLayer": "grafana",
```
Comment (m-s-) [inline cmf-cli/resources/template_feed/grafana/Grafana.Package/cmfpackage.json:6]: is there any case where a user must specify this? this is usually hidden from the developer as the package type has a specific target layer, and we add it in the package type handler. e.g. https://github.com/criticalmanufacturing/cli/blob/f2af42440c9f050e360b85ce1bd76127e6be35b3/cmf-cli/Handlers/PackageType/BusinessPackageTypeHandler.cs#L27

seems to me this is missing in the package handler, not here.

**Review by m-s- — CHANGES_REQUESTED** [review]

File: cmf-cli/resources/template_feed/grafana/Grafana.Package/cmfpackage.json (hunk 1 above)
Comment (obimelo) [reply cmf-cli/resources/template_feed/grafana/Grafana.Package/cmfpackage.json]: You are right.
It was an existing test case that led me in the wrong direction.
Check now please.

**Comment by joaoopereira** [comment]
@obimelo can you please follow our commit message convention?
https://www.conventionalcommits.org/en/v1.0.0/

this is what allow us to automatically generate changelogs

**Review by joaoopereira — DISMISSED** [review]

**Review by m-s- — APPROVED** [review]

**Comment by m-s-** [comment]
This needs to be rebased and the conflicts solved, I've done that in commit 59b5b32

**Pull request closed without merging**